jenkins/run-pytests

```diff
@@ -10,18 +10,22 @@ black --check .
 black_sts=${?}
 if [[ ${black_sts} -ne 0 ]]; then
     printf -- "FAILED: black --check .\n" >&2
+    (( errors++ ))
```

Comment:
You don't need to count the errors like this (and, you're missing the `typeset` declaration, which you should have if you _do_ want to do it this way), and, besides, your code is never checking the result of the count.  🙂

Instead, your code is capturing and saving the exit statuses and then adding them all together at line 30, which accomplishes basically the same thing, and that's fine.

So, either remove lines 13 and 21, or put them to some use.

Reply:
yes I missed the typeset, I will just remove the `error`